trade-finance-manager-api/api-tests/v1/amendments/create-amendment-tfm-object-on-amendment-completion.api-test.js

```diff
@@ -72,150 +67,167 @@ describe('update amendment-tfm on amendment completion', () => {
   });
 
   describe('createAmendmentTFMObject', () => {
-    describe('when TFM Facility end date feature flag is disabled', () => {
-      beforeEach(() => {
-        jest.mocked(isTfmFacilityEndDateFeatureFlagEnabled).mockReturnValue(false);
-      });
-
-      it('should not create tfm object when no getLatestCompletedAmendmentValue or getLatestCompletedAmendmentDate', async () => {
-        externalApis.getLatestCompletedAmendmentValue = jest.fn().mockResolvedValue({});
-        externalApis.getLatestCompletedAmendmentDate = jest.fn().mockResolvedValue({});
-        const result = await amendmentController.createAmendmentTFMObject(mockAmendment.amendmentId, mockAmendment.facilityId);
+    it('should not create tfm object when no getLatestCompletedAmendmentValue, getLatestCompletedAmendmentDate or getLatestFacilityEndDate', async () => {
+      externalApis.getLatestCompletedAmendmentValue = jest.fn().mockResolvedValue({});
+      externalApis.getLatestCompletedAmendmentDate = jest.fn().mockResolvedValue({});
+      externalApis.getLatestCompletedAmendmentFacilityEndDate = jest.fn().mockResolvedValue({});
 
-        expect(result).toEqual({});
-      });
+      const result = await amendmentController.createAmendmentTFMObject(mockAmendment.amendmentId, mockAmendment.facilityId);
 
-      it('should create tfm object when getLatestCompletedAmendmentValue and getLatestCompletedAmendmentDate', async () => {
-        externalApis.getFacilityExposurePeriod = jest.fn().mockResolvedValue({ exposurePeriodInMonths: 5 });
+      expect(result).toEqual({});
```

Comment:
is it possible to check that none of the calls made by createAmendmentTFMObject are called?

Reply:
This is all existing tests - I havent modified these at all & Im not familiar with how these work. The only change is a whitespace one. I can update this but it will take some time to familiarise myself & feels outside the scope of removing a feature flag